strawberryfields/backends/states.py

```diff
@@ -314,6 +365,7 @@ def poly_quad_expectation(self, A, d=None, k=0, phi=0, **kwargs):
         """
         raise NotImplementedError
 
+    @abc.abstractmethod
```

Comment:
Could this be added to `parity_expectation` too?

Reply:
Yep! Good catch